beacon-chain/node/node.go

```diff
@@ -658,6 +663,24 @@ func (b *BeaconNode) registerRPCService() error {
 
 	return b.services.RegisterService(rpcService)
 }
+func (b *BeaconNode) registerSubscriberRPCService() error {
+	config := &subscriber.Config{
+		IPCPath:    cmd.DefaultIpcPath,
```

Comment:
These endpoint values should be taken from flag.

Reply:
You're right. I won't change it for now, because we're moving to the gRPC smoothly.